.github/workflows/test.yml

```diff
@@ -5,7 +5,7 @@ jobs:
     strategy:
       matrix:
         #go-version: [1.15.x, 1.16.x]
-        go-version: [1.15.x]
```

Comment:
I wonder why this is not testing the 2 versions .... Is there a reason for it ? I.e. what's the point of building for the 2 versions if we just test one of them ?